src/components/Editor/Breadcrumb.js

```diff
@@ -0,0 +1,52 @@
+import { push } from '../../utils/router.js';
+
+export default function Breadcrumb({ $parent, initialState }) {
+  this.state = initialState;
+
+  const $breadcrumb = document.createElement('div');
+  $breadcrumb.classList.add('breadcrumb');
+  $parent.appendChild($breadcrumb);
+
+  this.setState = nextState => {
+    this.state = nextState;
+    this.render();
+  };
+
+  const createBreadCrumb = () => {
+    const breadcrumb = [];
+    if (this.state.id) {
+      let currentList = document.querySelector(
+        `li[data-id='${this.state.id}']`,
+      );
+      while (currentList !== null) {
+        const breadScrumbData = {};
+        breadScrumbData.id = currentList.dataset.id;
+        breadScrumbData.title = currentList.querySelector('p').textContent;
+        breadcrumb.push(breadScrumbData);
+        currentList = currentList.parentElement.closest('li[data-id]');
+      }
+    }
+    return breadcrumb.reverse();
+  };
+
+  this.render = () => {
+    $breadcrumb.innerHTML = `${
+      this.state.id
+        ? createBreadCrumb()
+            .map(
+              doc => `
```

Comment:
```suggestion
              ({ id, title }) => `
```
다음과 같은 형태로 사용하여서 사용할 인자만 선언해서 쓴다면 더 직관적으로 사용할 수 있습니다.